DuckDuckGo/OmniBar.swift

```diff
@@ -169,7 +169,13 @@ class OmniBar: UIView {
             return
         }
         resignFirstResponder()
-        omniDelegate?.onOmniQuerySubmitted(query)
+        
+        if let url = query.punycodedUrl {
```

Comment:
If the query can be coverted to a punycoded url it most likely means the user entered something like 💩.la or some other international domain name.  In that case we should use the punycoded version for processing, otherwise just use whatever text was entered and this will get treated like a query.